src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java

```diff
@@ -60,13 +66,13 @@ public String getSettingValue(String settingName, String defaultValue) {
         return settings.getProperty(settingName, defaultValue);
     }
 
-    private static void loadHttpsEnabled(PluginSettings instance, Properties settings) throws Exception {
+    private void loadHttpsEnabled() throws Exception {
         String httpsEnabledString = settings.getProperty(HTTPS_ENABLED, "False");
         if (httpsEnabledString == null) {
-            instance.httpsEnabled = false;
+            httpsEnabled = false;
         }
         try {
-            instance.httpsEnabled = Boolean.parseBoolean(httpsEnabledString);
+            httpsEnabled = Boolean.parseBoolean(httpsEnabledString);
         } catch (Exception ex) {
             LOG.error("Unable to parse httpsEnabled property with value {}", httpsEnabledString);
             instance.httpsEnabled = false;
```

Comment:
should this be "httpsEnabled = false" as well?

Reply:
Fixed.